python/Ganga/testlib/functions.py

```diff
@@ -0,0 +1,18 @@
+
+import random
+import tempfile
+import string
+
+def generateUniqueTempFile(ext = '.txt'):
```

Comment:
The rest of `testlib` is following PEP8-style naming for functions so `generate_unique_temp_file()` would fit better.